docs/src/pages/start/pick-quasar-flavour.md

```diff
@@ -21,22 +19,31 @@ Here's a quick comparison:
-| App icons & splash screens easy management through [Icon Genie](/icongenie/introduction) | - | **Yes** | - |
+| Easy management of App icons & splash screens via [Icon Genie](/icongenie/introduction)    | - | **Yes** | - |
 | Dynamic RTL support for Quasar components                                                  | **Yes** | **Yes**    | **Yes**                       |
 | Generating your own website/app RTL equivalent CSS rules automatically by Quasar           | -       | **Yes**    | **Yes**                       |
 | **Ensure everything "simply works" out of the box**, using latest and greatest Quasar specs.   | -       | **Yes**    | -                             |
 | **Tight integration between build modes**, taking full advantage of all Quasar's capabilities. | -       | **Yes**    | -                             |
-| Develop mobile apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes*                          |
-| One codebase to rule SPA, PWA, SSR, Mobile Apps, Electron Apps                             | -       | **Yes**    | Yes*                          |
+| One codebase to create SPA, PWA, SSR, Mobile Apps, Electron Apps                             | -       | **Yes**    | Yes(*)                      |
 | Tree Shaking                                                                               | -       | **Yes**    | **Yes**                       |
 | SFC (Single File Component - for Vue) support                                              | -       | **Yes**    | **Yes**                       |
 | Advanced configuration through dynamic quasar.conf.js                                      | -       | **Yes**    | -                             |
 | Unit & end to end testing                                                                  | -       | **Yes**    | **Yes**                       |
+| TypeScript support                                                              | -       | **Yes**    | **Yes**                       |
+|**Best and Most Popular Choice!**  |  |**Quasar CLI** |  |
 
-::: tip Footnote for above
-\* Cross-platform support with Vue CLI is handled by community plugins. These are not tightly integrated with Quasar as with Quasar CLI and so they may have issues. Also, Quasar CLI ensures Quasar is used at its best (including performance-wise).
+
+::: tip (*)Important!
+Although you may get a similar multi-platform support via the Vue CLI and some Vue community built plugins, these 3rd party supported build paths aren't tightly integrated with Quasar's components. Thus, as you run into problems with these 3rd party plugins, you will have to depend on the support of each individual plugin developer. With Quasar, you have a one-stop-shop should anything go wrong. Also, the Quasar CLI ensures applications are built to the best possible standards in both performance, project size and best practices. You will find no such guarantees anywhere else!
```

Comment:
Feels slightly off. Maybe:
"You may get a similar multi-platform support via the Vue CLI and some Vue community built plugins, but these 3rd party supported build paths aren't tightly integrated with Quasar's components."

Reply:
The sentence is grammatically correct. The "Although" is like a "but", just put at the front of the sentence to show from the start, the path of using the Quasar Plugin with Vue CLI is questionable. 

https://www.writewithjean.com/usage-and-grammar/although/

Scott

Reply:
I trust Scotts grammar so resolving this so we can merge :)